core/modules/telemetry/telemetry.telemetry.inc

```diff
@@ -18,6 +18,11 @@ function telemetry_telemetry_info() {
     'description' => t('The version number of your database (either MySQL or MariaDB).'),
     'project' => 'backdrop',
   );
+  $info['install_version'] = array(
+    'label' => t('Backdrop initial version'),
+    'description' => t('The Backdrop version at the time Backdrop was first installed, or 1.30.0 if it was installed before then.'),
```

Comment:
The update hook sets an estimated version, not "1.30.0", so this description is inaccurate.

```suggestion
    'description' => t('The Backdrop core version at the time Backdrop was first installed.'),
```